python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -464,6 +640,20 @@ def find_actual(target_name, fetch_list):
                             "Output (" + out_name + ") has different lod at " +
                             str(place) + " in dygraph mode")
 
+        if self.inplace_atol is not None:
+            warnings.warn(
+                "By default, inplace_atol should not be set, please check it")
+        self.check_inplace_output_with_place(place, no_check_set=no_check_set)
+
+        # TODO(zhiqiu): enhance inplace_grad test for ops (sum and activation) using mkldnn
+        if (hasattr(self, 'attrs') and self.attrs.get('use_mkldnn') is not None
+            ) or os.getenv('FLAGS_use_mkldnn') is not None:
```

Comment:
`FLAGS_use_mkldnn` may be False. Not suitable to check `os.getenv('FLAGS_use_mkldnn') is not None`.